packages/dpkg/build.sh

```diff
@@ -73,6 +74,7 @@ share/polkit-1
 termux_step_pre_configure() {
 	export TAR=tar # To make sure dpkg tries to use "tar" instead of e.g. "gnutar" (which happens when building on OS X)
 	perl -p -i -e "s/TERMUX_ARCH/$TERMUX_ARCH/" $TERMUX_PKG_SRCDIR/configure
+	sed -i 's/$req_vars = \$arch_vars.$varname./if ($varname eq "DEB_HOST_ARCH_CPU" or $varname eq "DEB_HOST_ARCH"){ print("'$TERMUX_ARCH'");exit; }; $req_vars = $arch_vars{$varname}/' scripts/dpkg-architecture.pl
```

Comment:
Latest build of dpkg's Makefile will regenerate configure file that make other patches of replacing arm64 to aarch64 invalidated, results in dpkg binary with native aarch "arm64" built. I patched the architecture script instead of messing with messing with the build script to let it use aarch64.